lib/db.js

```diff
@@ -54,7 +61,7 @@ function importBulkOsm (bbox, cb) {
     })
   }
   if (osmOrgDb) {
-    osmOrgDb.db.close(() => {
+    wipeDb(osmOrgDb.db, osmOrgDbPath, () => {
```

Comment:
Yeah we definitely do. This should be addressed in [1f809d7](https://github.com/osmlab/field-data-coordinator/pull/16/commits/1f809d715b6287297128a9ed2a4cfcfe6ddc94a8)